### PR TITLE
Changeset PRs

### PR DESCRIPTION
**Pull request by oveddan**
This adds a [github action from changesets](https://github.com/changesets/action) that versions the packages upon changes.

The way it works is:
* run `npx changeset` to generate a pending changeset.  Commit and push the files
* When this changeset is merged or pushed to `main`, a PR is created that applies the changeset and updates the corresponding package versions.  See [a sample PR here](https://github.com/ourzora/zora-1155-contracts/pull/97).  
* When that PR is merged, the versions are updated

When the `changeset` branch is merged to main, we auto-publish to npm.  See the [documentation here.](https://github.com/changesets/action)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    branches:
+      # todo: change to main once this is tested and merged
+      - changeset
```
Comment (oveddan) [inline .github/workflows/release.yml:7]: this should be changed to `main` when we are ready to merge

File: package/chainConfigs.ts [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-    MINT_FEE_RECIPIENT: "0x11cf5F667dC6AD4dEE58CB07e4AAc6a3fc7E1DCb",
-  },
-  [5]: {
-    FACTORY_OWNER: "0xDC498668B5e6CC518fD58A2ADBF614Fd3A13D3a0",
-    MINT_FEE_AMOUNT: 10000,
-    MINT_FEE_RECIPIENT: "0x9444390c01Dd5b7249E53FAc31290F7dFF53450D",
-  },
-};
+    [1]: {
+  "FACTORY_OWNER": "0xDB392f4391462d60B8B4413ef72018Ab595Af9D0",
```
Comment (iainnash) [inline package/chainConfigs.ts:3]: Why is this changed?

Comment (oveddan) [reply package/chainConfigs.ts]: shouldn't have been in this PR; I've updated to remove this delta.

**Review by iainnash — APPROVED** [review]
looks good :)
